src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/generate_params.py

```diff
@@ -299,6 +301,15 @@ def create_parameter_files(self) -> None:
                     ),
                     current_params
                 )
+                # Compare account_region final to global_stage
+                adf_org_stage = ADF_ORG_STAGE # Fetch from Environ for Start
+                current_params = self._merge_params(
+                    Parameters._parse(
+                        params_root_path=self.cwd,
+                        params_filename=f"global_{adf_org_stage}",
+                    ),
+                    current_params
+                )
```

Comment:
Let's use the global variable directly instead, so we don't need the
local variable and comment here:

```suggestion
                current_params = self._merge_params(
                    Parameters._parse(
                        params_root_path=self.cwd,
                        params_filename=f"global_{ADF_ORG_STAGE}",
                    ),
                    current_params,
                )
```